### ISSUE TITLE
Schedule tasks with no location

### ISSUE THREAD

**Issue by hawesie** [strands-project/strands_executive#41]
This one's for @mudrole1. I'd like to schedule tasks with no location. Currently when the start node id is empty, your code just passes this on through the `DistWrapper`. Really there should be no distance included for these tasks. Is that possible? There is also the problem of what happens to the tasks that come after it as they can no longer take their start location from the end location of the previous task.

**Comment by mudrole1** [comment]
Generally, having task with no location is possible. I got an idea how to fix the issue mentioned by you. I will try tomorrow to apply that.

**Comment by hawesie** [comment]
Cool, thanks.

**Comment by mudrole1** [comment]
We have discussed this with @BFALacerda and the most straightforward way is to take into account the worst travel distance. 

How important is this? I can prepare my scheduler, but Bruno is busy with more important work until Friday.

**Comment by bfalacerda** [comment]
@mudrole1  I just realized you can use the current dist function, call it for every pair of nodes, and calculate the max yourself. That would be as efficient as doing it on my side, at least until Dave Parker updates the prism API with some stuff I asked.

**Comment by hawesie** [comment]
@mudrole1 if you can do this on your side as suggested, that would be great for now.

**Comment by mudrole1** [comment]
Yep, I have done it. The updated code is in my repository.  @hawesie Will you merge it with STRANDS or shall I do it? only scheduler.cpp and scipUser.cpp + their header files have changed.

Now it takes into account the worst case scenario as we discussed. I am computing the maximal dist for places only occuring in the current scheduling problem (not all the places), but my computation is done always when scheduler is asked to solve given problem.  

I have tested it roughly, it seems working, at least some schedule is provided. I didnt get any proper idea how to break it.
